include/ck_tile/core/container/thread_buffer.hpp

```diff
@@ -169,4 +169,135 @@ struct vector_traits<thread_buffer<T, N>, std::enable_if_t<std::is_class_v<T>>>
 
 #endif
 
+template <char... Xs>
```

Comment:
thanks for adding printers! Eventually we may want to relocate them to a separate header for reusing